addons/point_of_sale/models/pos_session.py

```diff
@@ -643,18 +644,21 @@ def get_income_account(order_line):
         # The 'is_refund' parameter is used to compute the tax tags. Ultimately, the tags are part
         # of the key used for summing taxes. Since the POS UI doesn't support the tags, inconsistencies
         # may arise in 'Round Globally'.
+        check_refund = lambda x: (x.qty < 0 or x.price_unit < 0) and not (x.qty < 0 and x.price_unit < 0)
```

Comment:
I think it is simplier if we multiple qty and price_unit and check if the product is less than 0, like so:
`check_refund = lambda line: line.qty * line.price_unit < 0`